config/config.go

```diff
@@ -89,12 +104,10 @@ func init() {
 	cfg.SetDefault("analyzer.packet_injection_enabled", true)
 	cfg.SetDefault("analyzer.replication.debug", false)
 	cfg.SetDefault("analyzer.ssh_enabled", false)
-	cfg.SetDefault("analyzer.storage.bulk_insert", 100)
-	cfg.SetDefault("analyzer.storage.bulk_insert_deadline", 5)
-	cfg.SetDefault("analyzer.storage.index_age_limit", 0)
-	cfg.SetDefault("analyzer.storage.index_entries_limit", 0)
-	cfg.SetDefault("analyzer.storage.indices_to_keep", 0)
-	cfg.SetDefault("analyzer.storage.max_flow_buffer_size", 100000)
+	cfg.SetDefault("analyzer.storage.flow.bulk_insert", 100)
```

Comment:
please take a look at #968 
graph.backend => analyzer.topology.backend
analyzer.storage => analyzer.flow

analyzer.flow.{bulk_insert,bulk_insert_deadline,max_flow_buffer_size} must be removed from the configuration and deduced/resolved from the used backend (analyzer.flow.backend)

Reply:
ok, note that now:

analyzer.flow.{bulk_insert,bulk_insert_deadline,max_flow_buffer_size} ==> analyzer.storage.flow.{bulk_insert,bulk_insert_deadline,max_flow_buffer_size}